apps/project/schema.py

```diff
@@ -433,11 +441,24 @@ class Query:
             page_size_query_param='pageSize'
         )
     )
+    public_projects = DjangoPaginatedListObjectField(
+        ProjectListType,
```

Comment:
We need a custom type, this includes most of the project attributes.

We need to define specific fields.
```
                id
                title
                createdAt
                regions {
                    id
                    title
                }
                analysisFramework {
                    id
                    title
                }
                stats {
                    numberOfLeads
                    numberOfUsers
                }
                organizations {
                    id
                    organization {
                        id
                        title
                        mergedAs {
                            id
                            title
                        }
                    }
                }
 ```